File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/cn/porkchop/pinyougou/sellergoods/service/SellerService.java
package cn.porkchop.pinyougou.sellergoods.service;

import cn.porkchop.pinyougou.pojo.PageResult;
import cn.porkchop.pinyougou.pojo.TbSeller;

import java.util.List;

/**
 * 服务层接口
 *
 * @author Administrator
 */
public interface SellerService {

    /**
     * 返回全部列表
     *
     * @return
     */
    List<TbSeller> findAll();


    /**
     * 返回分页列表
     *
     * @return
     */
    PageResult findByPagination(int pageNum, int pageSize);


    /**
     * 增加
     */
    void add(TbSeller seller);


    /**
     * 修改
     */
    void update(TbSeller seller);


    /**
     * 根据ID获取实体
     *
     * @param id
     * @return
     */
    TbSeller findById(String id);


    /**
     * 批量删除
     *
     * @param ids
     */
    void delete(String[] ids);

    /**
     * 分页
     *
     * @param pageNum
     *         当前页 码
     * @param pageSize
     *         每页记录数
     * @return
     */
    PageResult findWithConditionsByPagination(TbSeller seller, int pageNum, int pageSize);

    /**
     * 更改审核状态
     *
     * @date 2018/6/23 13:20
     * @author porkchop
     */
    void updateStatus(String sellerId, String status);
}

File: pinyougou-parent/pinyougou-shop-web/src/main/webapp/js/service/sellerService.js
//服务层
app.service('sellerService', function ($http) {

    //读取列表数据绑定到表单中
    this.findAll = function () {
        return $http.get('../seller/findAll.php');
    };
    //分页
    this.findByPagination = function (page, rows) {
        return $http.get('../seller/findByPagination.php?page=' + page + '&rows=' + rows);
    };
    //查询实体
    this.findById = function (id) {
        return $http.get('../seller/findById.php?id=' + id);
    };
    //增加
    this.add = function (entity) {
        return $http.post('../seller/add.php', entity);
    };
    //修改
    this.update = function (entity) {
        return $http.post('../seller/update.php', entity);
    };
    //删除
    this.dele = function (ids) {
        return $http.get('../seller/delete.php?ids=' + ids);
    };
    //搜索
    this.findWithConditionsByPagination = function (page, rows, searchEntity) {
        return $http.post('../seller/findWithConditionsByPagination.php?page=' + page + "&rows=" + rows, searchEntity);
    };
    //读取列表数据绑定到表单中
    this.getUserName = function () {
        return $http.get('../seller/getUserName.php');
    };
});
